apps/mosaico/src/Mosaico.purs

```diff
@@ -360,13 +360,17 @@ routeListener c setState _oldLoc location = do
   runEffectFn1 refreshAdsImpl ["mosaico-ad__top-parade", "mosaico-ad__parade"]
 
   case match (Routes.routes c) $ Routes.stripFragment $ location.pathname <> location.search of
-    Right path -> setState \s -> s { route = path
-                                   , prevRoute = Just s.route
-                                   , clickedArticle = case path of
-                                       Routes.ArticlePage articleId
+    Right path -> do
+      setState \s -> s { route = path
```

Comment:
Could you please edit this? The left alignment is all out of whack in this section.